tools/build/workspace.bzl

```diff
@@ -158,6 +158,11 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
         build_file = "@gapid//tools/build/third_party:lss.BUILD",
     )
 
+    native.local_repository(
+        name = "perfetto",
+        path = "/usr/local/google/home/chiur/dev/perfetto"
```

Comment:
Well this doesn't seem right,

Reply:
Right, I am waiting for some perfetto changes.  But I want to put this up for review because this is the first time I am modifying the layer and not sure if this is how things are expected to be done.

Reply:
Fixed, PTAL